pontoon/base/models.py

```diff
@@ -469,6 +469,16 @@ def get_top_instances(self):
 
 class Locale(AggregatedStats):
     code = models.CharField(max_length=20, unique=True)
+    db_collation = models.CharField(
+        max_length=20,
+        blank=True,
+        help_text="""
+        Some of locales require to use different database collation than default ('en_US').
+
+        <strong>Use with caution, because it may brake the search for this locale.</strong>
```

Comment:
```Some locales require different database collation than default ('en_US').

<strong>Use with caution, because it may break search for this locale.</strong>
```